rust-runtime/aws-smithy-types/src/retry.rs

```diff
@@ -5,12 +5,12 @@
 
 //! This module defines types that describe when to retry given a response.
 
-use std::borrow::Cow;
-use std::fmt::{Display, Formatter};
-use std::num::ParseIntError;
+use std::fmt;
 use std::str::FromStr;
 use std::time::Duration;
 
+const VALID_RETRY_MODES: &[RetryMode] = &[RetryMode::Standard];
```

Comment:
```suggestion
const SUPPORTED_RETRY_MODES: &[RetryMode] = &[RetryMode::Standard];
```
I think "supported" is a better word for this